### PR TITLE
Gamecube, OneDrive option and fix for ePSXe included

### PR DESCRIPTION
**Pull request by PRElias**
Hi Chris

I dont know if you are interested, but I included the dolphin Gamecube emulador

### CODE REVIEW

**Comment by Francommit** [comment]
Hey @PRElias ,

Just had a quick look through seems to be ok. Though in the spirit of full automation on this page in your link the only thing I noticed being required was:

> You should now have the required game INI settings placed in the proper directory. Dolphin will look inside this directory for game-specific recommended settings.

If this is the case. We should generate the .ini ourselves so users don't have to configure anything themselves.

**Comment by PRElias** [comment]
Hi,

Yes, but as stated on the link that I provided, Retroarch team is planning to make that config default on future versions.

By the way, I agree with you and I will look for that.

Thanks

**Comment by PRElias** [comment]
Hi Chris,

I created the section for the ini file, tested, and included a tip on configuring Xbox controllers

**Comment by PRElias** [comment]
Chris,

Sorry, but I am not an expert on GitHub as well. I thought that only my commits made before submitting the PR will be in the package, but looking at the PR again, I guess I'm wrong.

Then, I will analyze each of them, specially that one on automatize chocolatey installation, based on your great feedback

**Comment by PRElias** [comment]
I reverted the commit on Choco auto-install based on your comment and I learnt that to keep my new commits separate from this PR, I would have created a new branch. Sorry about that.

I guess I finally realized what are you saying about admin privileges on script. But, the commands to install choco does not try to set that and we already ask for use an admin session...

File: prepare.ps1 [hunk 1]
```diff
@@ -1,15 +1,6 @@
-# Configuring
-Set-ExecutionPolicy -ExecutionPolicy Bypass -Force
-
 Import-Module BitsTransfer
 [Net.ServicePointManager]::SecurityProtocol = "tls12, tls11, tls" #Convince Powershell to talk to sites with different versions of TLS
 
-# Installing Chocolatey 
-Invoke-Expression ((new-object net.webclient).DownloadString('https://chocolatey.org/install.ps1'))
```
Comment (Francommit) [inline prepare.ps1]: if you want to test this bit and it works happy for you to keep it in :)

File: README.md [hunk 2]
```diff
@@ -20,10 +20,12 @@ Translations
 
 Steps
 ------
-1. Run prepare.ps1 in an admin session of Powershell
+1. Ensure chocolatey is installed https://chocolatey.org/install
```
Comment (Francommit) [inline README.md]: If you're happy with your PR in this state i'll give it a test unless you want to chance some more things!

File: prepare.ps1 (hunk 1 above)
Comment (PRElias) [reply prepare.ps1]: Hi,

I have tested on my notebook (uninstalling chocolatey first). But i'll do some more tests and ask friends to do that too. By now, I moved this feature to another branch and I intend to submit a new PR if I have sucess on them.

That being said, if you want to accept PR as it is now (without the auto-installer for choco), I think it's safe

File: README.md (hunk 2 above)
Comment (PRElias) [reply README.md]: Hi,

As I said, I learnt now how to make a "safe PR" using branches, so I will run more tests and submit again in the future on a new PR, ok

File: prepare.ps1 (hunk 1 above)
Comment (Francommit) [reply prepare.ps1]: hey when you're ready to submit the other PR i'll test that. bit busy with work atm. Thanks heaps PRElias!

**Comment by Francommit** [comment]
@PRElias this PR isn't required anymore is it? It looks like this one contains the commit that are now in master. I don't know why Github's still saying this one exists?

**Comment by PRElias** [comment]
I think so, sorry for the mess. I am not enough experienced in Github and caused this confusion. You can dismiss this one and I will prepare a new PR if is it necessary to correct something. Sorry again.

**Comment by PRElias** [comment]
Probably this problem was caused by me making changes on master...I'll avoid that in the future

File: prepare.ps1 [hunk 3]
```diff
@@ -782,7 +766,7 @@ Set-ItemProperty -Path $path -Name 'CPUOverclocking' -Value '10'
 # 
 # 17. Add in a game art scraper
 # 
-$scraperZip = $requirementsFolder + "scraper_windows_amd64*.zip"
+$scraperZip = $requirementsFolder + "scraper_windows_amd64-v1.4.5.zip"
```
Comment (Francommit) [inline prepare.ps1]: Isn't this the incorrect version?

Comment (PRElias) [reply prepare.ps1]: The correct version is the one with the *. You'll have to exclude this PR because it was made before the other one.

**Comment by ghost** [comment]
Hello can you guys please include Dreamcast and Naomi stuff? also PSP plz

**Pull request closed without merging**